apps/src/dropletUtils.js

```diff
@@ -276,6 +287,18 @@ exports.generateDropletModeOptions = function (dropletConfig) {
   populateModeOptionsFromConfigBlocks(modeOptions, { blocks: exports.dropletBuiltinConfigBlocks });
   populateModeOptionsFromConfigBlocks(modeOptions, dropletConfig);
 
+  setTitlesToFuncNamesForDocumentedBlocks(modeOptions);
+
   return modeOptions;
 };
 
+/**
+ * Returns a set of all blocks
+ * @param dropletConfig
+ * @returns {Object.<String, Object>}
+ */
+exports.getAllDropletBlocks = function (dropletConfig) {
+  return $.extend(exports.dropletGlobalConfigBlocks,
+    exports.dropletBuiltinConfigBlocks,
+    dropletConfig.blocks);
+};
```

Comment:
Good catch, didn't mean to modify dropletGlobalConfigBlocks, adding a `{},` at the beginning there